reana_client/api/client.py

```diff
@@ -1018,3 +1018,28 @@ def list_secrets(access_token):
         raise Exception(e.response.json()["message"])
     except Exception as e:
         raise e
+
+
+def workspaces(access_token):
+    """List available workspaces."""
```

Comment:
Minor: `:param access_token: access token of the current user.` is missing from the docstring